internal/rules/aws/ec2/no_excessive_port_access.go

```diff
@@ -39,7 +39,7 @@ var CheckNoExcessivePortAccess = rules.Register(
 	func(s *state.State) (results scan.Results) {
 		for _, acl := range s.AWS.EC2.NetworkACLs {
 			for _, rule := range acl.Rules {
-				if rule.Protocol.EqualTo("-1") || rule.Protocol.EqualTo("all") {
+				if rule.Type.EqualTo("egress") && rule.Action.EqualTo("allow") && rule.Protocol.EqualTo("-1") || rule.Protocol.EqualTo("all") {
```

Comment:
I thought we should confirm `rule_action` is `allow`.

Reply:
we are confirming the rule_action is allow. I added egress to align with the check.

Reply:
Yeah, I saw it. We should discuss ingress/egress. I'll mark it as resolved.